### DISCUSSION TITLE
Base assets timeframe

### DISCUSSION THREAD

**Discussion by 12gabriel3** [EveronLife/EveronLife#133] [Polls]
We had this discussion while making emergency lights for police and other emergency services.

Many objects are time dependent, especially electronics such as cellphones, radios, TVs to name a few.
The server owners can always choose, but we are providing some assets with the framework.

I think with this poll, we should have a more solid guidance when choosing the assets styles.

**Comment by 12gabriel3** [comment]
I think game timeframe fits better with the other available assets. If we want to provide a framework, we should stick to vanilla style as much as possible. A server owner can always make themed servers ie. present or future.

**Reply by CapValGo** [reply]
In support of 80s time frame, we already have useable materials/textures that fit with the current feel.

**Comment by Tomboru** [comment]
As I said: 

I would say modern.
Even if AR is in the past.

If the Framework gets ported to Arma4 it would maybe fit better the time
AR was just a "remake" of cold war as we know.
I think the time of Arma 4 will be similar to Arma 3, maybe 20-30 years more in the future

**Comment by HarrisOik** [comment]
I believe that somewhere between Modern (Present) and Futuristic would be the best since (as far as we know) Arma 4 will probably take place in the near future (about 20-25 years, similar to Arma 3). Of course we cannot know the actual setting until details are revealed, but going Modern-Futuristic would be the better option here.

**Comment by CapValGo** [comment]
Personally, I'd say time period.

It's life in everon, which has a 80s feel, seems a bit silly, to make a framework for "US life" with straight roads, when the server owner would have to make a terrain.

I personally tried to make a life server when (GM) Werferlingen came out.

**Comment by TheOriginalCrazyDuck** [comment]
I would say Modern, while it's a neat idea to have 80s assets, you'd also be dealing with 80s limitations and I feel it would make for a stronger base to have modern assets which server owners can swap out/disable as their desired time period suits.

**Reply by HarrisOik** [reply]
I do agree with the limitations part. Even in something trivial like emergency services lights, we are limited to what operations they can have i.e. modern police cruisers have traffic split modes on their lightbars (similar to scrolling indicators found on modern cars) to direct traffic away (indicating left or right) from a traffic stop or a car accident. This would be impossible without a modern lightbar.

**Comment by jasper-one** [comment]
I strongly believe it should be set on the reforger timeline which is 1989 so much closer to the 90s then the 80s.

To be clear, the entire game and all of its assets are set to then. So to not have the immersion removed for another time period you basically have to rebuild every asset and the entire map from scratch.

Not to mention we want to try and heavily leverage existing assets to keep this framework small to work on consoles so the majority of assets will be from that time period anyway. 

In 1989 Windows existed for 4 years at that point, you had the Macintosh, the first mobile phone was invented. There's a bunch of time period accurate tech that will more then support every life concept that Altis Life and others had.

We shouldn't be recontemplating the assets time period based on a single light bar for a cop car or the fact that Arma 4 is being released in a number of years.  We can simply switch the assets then and leverage existing ones within Arma 4.

Ofcourse not every single thing needs to be exactly time period accurate but we should keep the aesthetic and be accurate where possible.

**Reply by Arkensor** [reply]
Yep, this. We should try to make as few models as possible and if we add any we should try to make them timeless if possible. Lightbars on police vehicles are a bit tough in this regard. I would not make them obvious to be from the 80's, but also not a massive LED bar that won't fit on the vanilla vehicles very well.

In general, I think the setting should be neutral if possible. If it has to be either old or modern I think we might want to go with old for now (stuff like smartphones vs old bulky phones). For Arma 4 we will rebuild a lot of things most likely and fix any architecture problems we learned about in AR. Swapping out models for a more modern setting there will not be a big problem. So I would only think about AR for now.
